test/Microsoft.AspNet.Mvc.Core.Test/ControllerTests.cs

```diff
@@ -658,6 +658,21 @@ public void HttpNotFound_SetsStatusCode()
         }
 
         [Fact]
+        public void HttpNotFound_SetsStatusCodeAndResponseContent()
+        {
+            // Arrange
+            var controller = new TestableController();
+
+            // Act
+            var result = controller.HttpNotFound("Test Content");
+
+            // Assert
+            Assert.IsType<HttpNotFoundObjectResult>(result);
+            Assert.Equal(StatusCodes.Status404NotFound, result.StatusCode);
+            Assert.Equal("Test Content", result.Value);
+        }
```

Comment:
Added a test to `HttpNotFoundObjectResultTest` that verifies execute behavior. Since this is just controller unit tests I don't see the point to testing the same thing here.